examples/ccenc.rs

```diff
@@ -223,13 +224,14 @@ fn new_av1_vaapi_encoder(
         height: args.height,
     };
 
-    let mut config = av1::EncoderConfig {
+    let config = av1::EncoderConfig {
         resolution,
         ..Default::default()
     };
 
+    let mut tunings = Tunings::default();
     if let Some(framerate) = args.framerate {
-        config.framerate = framerate;
```

Comment:
This makes me think that maybe we can embed the `Tunings` in the `EncoderConfig`, as they seem to always be passed in pairs? This would reduce the number of arguments passed to `new_vaapi`/`new_av1`, and hammer the fact that in this context the tunings are part of the initial configuration.

Reply:
Yeah, you are right. I wanted to avoid the situation when values from tunings were used by mistake. I've added the field and named it `initial_tunings` to make sure it would not be misused.